style/components/badge.css

```diff
@@ -0,0 +1,99 @@
+.badge-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 12rem;
+  border-bottom: 1px solid rgb(156, 156, 156);
+  padding: 10px;
+  margin-bottom: 10px;
+}
+
+.txt-badge {
+  background-color: #0d8dc9;
+  padding-left: 0.6rem;
+  padding-right: 0.6rem;
+  padding-top: 0.3rem;
+  padding-bottom: 0.3rem;
+  border-radius: 50%;
+  position: absolute;
+  top: -8%;
+  right: -2%;
+  color: white;
+  font-size: 12px;
```

Comment:
Try to use one specific CSS unit for overall project.

Reply:
Okay I will keep this in mind